src/tsconfig.json

```diff
@@ -1,18 +1,19 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2021",
```

Comment:
chore: maybe we we don't need that anymore now that we migrated the code to node 18 and yarn 3. Let's try to remove this.

Reply:
Not harmful and it's an improvement. Let's do it.